frb/builds/build_frbs.py

```diff
@@ -121,27 +130,37 @@ def frb_181112():
 
 def frb_190102():
     """Bhandari+20, ApJL --
+    Now including updated values on FRB position, DM, pulse width 
+    and scattering time from Day et al. 2020
     """
     fname = 'FRB190102'
-    wv_oiii = 6466.48
-    z_OIII = wv_oiii / 5008.239 - 1
-    frb190102 = frb.FRB(fname, 'J212939.7-792832.3', # Pulled from Slack 26-03-2019
-                        362 * units.pc / units.cm**3,
-                        z_frb=z_OIII)
+    #wv_oiii = 6466.48
+    #z_OIII = wv_oiii / 5008.239 - 1
+    frb190102 = frb.FRB(fname, 'J212939.76-792832.5', # Day+20
+                        364.545 * units.pc / units.cm**3,
+                        z_frb=0.2912) # Updated redshift
     # Error ellipse [REQUIRED]
-    frb190102.set_ee(0.1, 0.1, 0., 95.)
+    frb190102.set_ee(0.1, 0.1, 0., 95.) #statistical
+    frb190102.set_ee(0.4, 0.5, 0., 95., stat=False) #systematic
 
     # Error in DM
-    frb190102.DM_err = 1 * units.pc / units.cm**3
+    frb190102.DM_err = 0.004 * units.pc / units.cm**3
 
     # NE2001
     frb190102.set_DMISM()
-    # RM
-    #frb190102.RM = 10 * units.rad / units.m**2
-    #frb190102.RM_err = 1 * units.rad / units.m**2
-
+    
+     # FRB properties
+    frb190102.fluence = 14 * units.Jy * units.ms    # From Bhandari+20
+    frb190102.fluence_err = 1 * units.Jy * units.ms # -- //-- 
+    frb190102.RM = -105 * units.rad / units.m**2
+    frb190102.RM_err = 1 * units.rad / units.m**2
+    frb190102.width = 0.053 * units.ms
```

Comment:
There are hooks already for Width.  See `frb.FRB`
We need to have a uniform approach to this.